### DISCUSSION TITLE
Flickering inscriptions ST7796 stm32f411

### DISCUSSION THREAD

**Discussion by k008** [Bodmer/TFT_eSPI#1621] [Q&A - Fonts]
Hello, tell me why almost all fonts flicker on ST7796 stm32f411?
But it does not flicker if you use the font indicated by the number:
`tft.drawString(String(0) + String(tm.Second), 380, 200, 6); `

**Comment by Bodmer** [comment]
This is down to the redraw speed, fonts 2, 4, 6, 7 and 8 were created for the UNO and Mega so are optimised for speed using Run Length Encoding. Flicker is therefore minimised.

Sprite are the best way to avoid flicker.


The b

**Comment by Bodmer** [comment]
This library uses a neat trick to reduce flicker with the Adafruit Free Fonts:
https://github.com/KrisKasprzak/FlickerFreePrint

**Comment by k008** [comment]
What causes flickering?
What is the best way to use Sprite, for each group of captions or for the whole screen?
The screen size is 320x240, I want to display pictures of car icons on this screen and sign the values ​​(engine temperature, oil temperature, current time, air temperature in the car and outside, inclinometer)
